in-vehicle-digital-twin/src/digitaltwin_impl.rs

```diff
@@ -34,36 +32,23 @@ impl DigitalTwin for DigitalTwinImpl {
         request: Request<FindByIdRequest>,
     ) -> Result<Response<FindByIdResponse>, Status> {
         let request_inner = request.into_inner();
```

Comment:
since we are only using `request_inner` to access the `id` property, we can elide this to `let entity_id = request.into_inner().id;`

Reply:
Fixed.